src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -960,6 +965,9 @@ protected function getColumnSqlDefinition(Column $column)
         } elseif (isset($sqlType['limit'])) {
             $def .= '(' . $sqlType['limit'] . ')';
         }
+        if(($values = $column->getValues()) && is_array($values)) {
```

Comment:
Still slightly off for PSR compatibility, but close enough. (Need `if (` with a space.)